adapters/improvedigital/improvedigital.go

```diff
@@ -222,3 +256,37 @@ func (a *ImprovedigitalAdapter) getAdditionalConsentProvidersUserExt(request ope
 
 	return extJson, nil
 }
+
+func getImpExtWithRewardedInventory(imp openrtb2.Imp) ([]byte, error) {
+	var ext = make(map[string]json.RawMessage)
+	if err := json.Unmarshal(imp.Ext, &ext); err != nil {
+		return nil, err
+	}
+
+	prebidJSONValue, prebidJSONFound := ext["prebid"]
+	if !prebidJSONFound {
+		return nil, nil
+	}
+
+	var prebidMap = make(map[string]json.RawMessage)
+	if err := json.Unmarshal(prebidJSONValue, &prebidMap); err != nil {
+		return nil, err
+	}
+
+	rewardedInventory, foundRewardedInventory := prebidMap[isRewardedInventory]
+	if !foundRewardedInventory {
+		return nil, nil
```

Comment:
I see this line isn't covered by tests. Is it possible to test this case?

Reply:
Test case added